.semaphore/semaphore.yml

```diff
@@ -18,17 +20,15 @@ blocks:
         - name: Build Docker Image
           commands:
             - make docker/build
-            - make docker/save
-            - cache store ${DOCKER_IMAGE_TAG} ${DOCKER_IMAGE_TAG}.tar
+            - make semaphore/store
```

Comment:
semaphore is not the right namespace. The whole point of the makefile is to be cloud-agnostic. The Job has nothing to do with semaphore specifically but can be applied on to various CIs. We should rename to either docker/store or ci/docker-save, ci/docker-load, or so

Reply:
cache is a tool which is only available within semaphore CI...